modules/@apostrophecms/submitted-draft/index.js

```diff
@@ -23,12 +23,12 @@ module.exports = {
         component: 'AposCellType'
       },
       'submitted.at': {
-        label: 'Submitted',
-        component: 'AposCellDate'
+        label: 'Submitted'
+        // component: 'AposCellDate'
       },
       'submitted.by': {
-        label: 'Proposed By',
-        component: 'AposCellDate'
+        label: 'Proposed By'
+        // component: 'AposCellDate'
```

Comment:
This one should never have used AposCellDate. The other one should, not sure why that's not an option at the moment.

Reply:
Oh, I didn't mean to leave these commented out.

Reply:
There's a separate ticket to work on those. It's because they aren't digging into the data property with the dot path.